extruct/tool.py

```diff
@@ -32,6 +35,14 @@ def metadata_from_url(url, microdata=True, jsonld=True, rdfa=True):
         rdfae = RDFaExtractor()
         result['rdfa'] = rdfae.extract_items(tree, resp.url)
 
+    if opengraph:
+        oge = OpenGraphExtractor()
+        result['opengraph'] = [obj for obj in oge.extract_items(tree, resp.url)]
+
+    if microformat:
+        mfmate = MicroformatExtractor()
+        result['microformat'] = [obj for obj in mfmate.extract_items(html=resp.content, url=resp.url)]
```

Comment:
Could this code be simplified now, as there is a top-level extraction function which allows to choose which formats to use?